Scripts/AnalyzeDatabank/calcOrderParameters.py

```diff
@@ -15,8 +12,8 @@
 ### This is the NMRlipids databank repository path
 databankPath = '../../'
 
-sys.path.insert(1, databankPath + '/Scripts/BuildDatabank/')
-from databankLibrary import *
+sys.path.append('..')
+from DatabankLib.databankLibrary import *
```

Comment:
to have an internal consistency in importing databank related modules and methods, could you please explicitly define the imported methods here, instead of using *?

Reply:
done during style alignment.